packages/runtime/src/modules/RequestModule.ts

```diff
@@ -134,6 +128,9 @@ export class RequestModule extends RuntimeModule {
                     response: responseWrapper.response
                 });
                 break;
+            case "ArbitraryMessageContent":
+            case "Mail":
+            case "Notification":
```

Comment:
please add breaks for all of these cases or re-add default and add a break for this. This was my intention for the comment i had before.